domain-models-runtime/src/main/resources/templates/db_scripts/indexes.ftl

```diff
@@ -3,7 +3,7 @@
     <#list table.index as indexes>
     <#if indexes.tOps.name() == "ADD">
     CREATE INDEX IF NOT EXISTS ${table.tableName}_${indexes.fieldName}_idx ON ${myuniversity}_${mymodule}.${table.tableName}
-    (${indexes.getFinalSqlExpression(table.tableName)})
+    (left(${indexes.getFinalSqlExpression(table.tableName)},600))
```

Comment:
This is for the non-unique b-tree index only.

Please also add this for the unique b-tree.

Reply:
I do not understand what needs to be done here.  Can you explain what the difference is?

Reply:
left(..., 600) is missing in line 19.

Reply:
it appears that there is already a unit test checking for this index inside of SchemaMakerTest.ginIndex.  It failed the test after I modified it